i18n/en.json

```diff
@@ -367,6 +367,14 @@
     "id": "api.channel.rename_channel.cant_rename_group_messages.app_error",
     "translation": "You cannot rename a group message channel"
   },
+  {
+    "id": "api.channel.restore_channel.restored.app_error",
+    "translation": "Unable to unarchived a not archived channel"
```

Comment:
```suggestion
    "translation": "Unable to unarchive channel. The channel is not archived."
```

Reply:
^proposed update. Definitely open for feedback, I'm only 1/5 on this particular wording. @wiersgallak wondering if you had any feedback to share,

Reply:
i like this better than the one before. went with this change